### DISCUSSION TITLE
Do you have plans to update the reframework to work on the full release of RE4 Remake?

### DISCUSSION THREAD

**Discussion by CallMeDende** [praydog/REFramework#684] [General Questions] [answered]
It worked perfectly on the Demo but after installing the DLSS mod the game almost crashes immediately after booting it up.

**Comment by praydog** [comment, marked as answer]
Yes. It's not out for me yet.

**Reply by praydog** [reply]
Looks like flying to New Zealand did the trick.

**Reply by J4cKP0t777** [reply]
u in NZ?? :)

**Comment by CallMeDende** [comment]
Eyy I used the same trick to unlock the game early using a VPN:) hope to see the reframework up and running again soon.


 
 
  On Thu, Mar 23, 2023 at 9:21 PM, ***@***.***> wrote:   


Looks like flying to New Zealand did the trick.
